src/vc_generic.c

```diff
@@ -101,6 +101,7 @@ clnt_read_vc(XDR *xdrs, void *ctp, void *buf, int len)
 	return (len);
 }
 
+#if 0
```

Comment:
The whole file will be nuked in a future patch. #if 0 was used here for ease of reviewing, and ease of looking back at the old code during development.